src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -169,15 +163,22 @@ private Map<EventKey, Event> parseEventRecordsToEventsEntityMap(Set<String> even
                   "Ensuring orgId={} has been set up for syncing/reporting.", event.getOrgId());
               ensureOptIn(event.getOrgId());
             }
-            eventsMap.putIfAbsent(EventKey.fromEvent(event), event);
+
```

Comment:
Are you speaking about this line https://github.com/RedHatInsights/rhsm-subscriptions/blob/b9e26babf81fdc59a2aa1cbc3e1ccbfdd4e884f3/src/main/java/org/candlepin/subscriptions/event/EventController.java#L160 ? 
If so, I agree with you that this causes a lot of noise, but I didn't change it as part of this pull request, so it was originally like this.

Reply:
I'm suggesting that we change logging based on source_type because the Prometheus data will generate a lot of noise following this PR. The original purpose of this logging was for cost management since we have no method of getting the same message again. So we should keep it but only for `event_source: cost-management`

Reply:
`logging` is too generic to me. 

if you mean to do this:
https://github.com/RedHatInsights/rhsm-subscriptions/blob/b9e26babf81fdc59a2aa1cbc3e1ccbfdd4e884f3/src/main/java/org/candlepin/subscriptions/event/EventController.java#L160
Only when `event_source: cost-management`, then I'm ok with it.